tests/Doctrine/Tests/DBAL/ConnectionTest.php

```diff
@@ -297,6 +298,58 @@ public function testCommitStartsTransactionInNoAutoCommitMode() : void
         self::assertTrue($conn->isTransactionActive());
     }
 
+    /**
+     * @dataProvider resultProvider
+     */
+    public function testCommitReturn(bool $expectedResult) : void
+    {
+        $pdo = $this->createMock(PDO::class);
+        $pdo->expects($this->once())
+            ->method('commit')->willReturn($expectedResult);
+
+        $driverMock = $this->createMock(Driver::class);
+        $driverMock->expects($this->any())
+            ->method('connect')
+            ->will($this->returnValue($pdo));
```

Comment:
Why `PDO`? https://github.com/doctrine/dbal/blob/0152fbf4d72a7889996bfd7dc7ccc4bc678f4a15/lib/Doctrine/DBAL/Driver.php#L22-L24

Reply:
Fixed. Thank you.